### PR TITLE
grt: shuffle congestion markers for better visualization on gui

### PR DESCRIPTION
**Pull request by eder-matheus**
Fix https://github.com/The-OpenROAD-Project/OpenROAD/issues/6386

### CODE REVIEW

File: src/grt/src/fastroute/src/utility.cpp [hunk 1]
```diff
@@ -2488,6 +2490,13 @@ void FastRouteCore::saveCongestion(const int iter)
   std::vector<CongestionInformation> congestionGridsV, congestionGridsH;
   if (!h_edges_.empty() && !v_edges_.empty()) {
     getCongestionGrid(congestionGridsV, congestionGridsH);
+
+    std::mt19937 g;
+    const int seed = 42;
+    g.seed(seed);
+
+    utl::shuffle(congestionGridsV.begin(), congestionGridsV.end(), g);
+    utl::shuffle(congestionGridsH.begin(), congestionGridsH.end(), g);
```
Comment (maliberty) [inline src/grt/src/fastroute/src/utility.cpp:2499]: There is still an ordering for V markers then H markers.  With many markers the V will hit the limit and block the H.

Comment (eder-matheus) [reply src/grt/src/fastroute/src/utility.cpp:2499]: Is that right? When I open the GUI, I see two different tabs for V and H, and both of them have 10000 occurrences.

Comment (maliberty) [reply src/grt/src/fastroute/src/utility.cpp:2499]: I didn't realize they are separate categories

**Comment by maliberty** [comment]
grt.congestion7.tcl (Failed)

**Review by maliberty — APPROVED** [review]

**Pull request merged**